lib/urlwatch/config.py

```diff
@@ -68,6 +68,7 @@ def parse_args(self, cmdline_args):
         parser = argparse.ArgumentParser(description=urlwatch.__doc__,
                                          formatter_class=argparse.RawDescriptionHelpFormatter)
         parser.add_argument('joblist', metavar='JOB', type=int, nargs="*", help='index of job(s) to run, as numbered according to the --list command. If none specified, then all jobs will be run.')
+        parser.add_argument('--tags', metavar='TAG,TAG...', type=lambda x: x.split(","), help='comma separated tags of job(s) to run. If none specified, then all jobs will be run.')
```

Comment:
Shouldn't this be using [`nargs="+"`](https://docs.python.org/3/library/argparse.html#nargs), possibly with `action="extend"`?

Reply:
We already take a variable number of job indexes, so I thought nargs + would be confusing. How would you interpret `urlwatch --tags abc def 1 6`?

We could potentially say you can't use them both together, and that `--tags` changes from using job indexes to tags on the command line. Wdyt?

Reply:
Yeah, making `--tags` an `action="store_true"` and treating `joblist` as a list of tags if `args.tags` is `True` seems like a sensible idea. It then basically becomes a switch that changes the meaning of the positional arguments from job indexes to tags (the `joblist` help string might also need updating then).

Reply:
I've done that, and refactored to clean it up a little. Squashed the push because it was a fairly heavy change to the approach.